SR2024/2023-12-13-january-tech-day-confirmation.md

```diff
@@ -0,0 +1,18 @@
+---
+to: Teams accepted for the January Tech Day
+subject: January Tech Day confirmation
+---
+
+Hi $TODO_CONTACT_GIVEN_NAME,
+
+I'm pleased to confirm your place at the January 20th Tech Day in Horsham. 
+
+The venue has a small car park to the rear which you are welcome to use. If you do plan to use this, please let us know the registration number of the vehicle so we can add it to their parking system.
+
+There's an outline of the day on the event page on our website. Exactly what your team work on during the day is up to them -- we don't have any set schedule. Some teams use Tech Days as an opportunity to generally work on their robot, while others focus on a specific task they want to achieve or get help with.
```

Comment:
Do we want to link to the event page? And/or the robots-101 Tech Day page in the docs?
```suggestion
There's an outline of the day on the [event page][event-page] on our website. Exactly what your team work on during the day is up to them -- we don't have any set schedule. Some teams use Tech Days as an opportunity to generally work on their robot, while others focus on a specific task they want to achieve or get help with.

[event-page]: https://studentrobotics.org/events/sr2024/horsham-tech-day-january/
```